docs/excel/custom-functions-overview.md

```diff
@@ -360,6 +360,50 @@ function secondHighest(values){
 }
 ```
 
+## Discovering cells that invoke custom functions
+
+You may wish to discover the cell that invoked a custom function. Common scenarios for using `caller.address` include:
+
+- Formatting ranges: You can use `caller.address` as the key of the cell to store information in [AsyncStorage](https://docs.microsoft.com/office/dev/add-ins/excel/custom-functions-runtime#storing-and-accessing-data). Then, use [onCalculated](https://docs.microsoft.com/javascript/api/excel/excel.worksheet#oncalculated) in Excel to load the key from `AsyncStorage`.
```

Comment:
Consider eliminating the "you cans" (See "our voice is human" in the Style Guide: https://microsoft.sharepoint.com/teams/BrandCentral/Pages/The-Microsoft-brand-Identity-Voice.aspx)

Reply:
Eliminate "passive voice", should be "If your function is used offine, display stored cached values from..." (Not "is being used")

Reply:
Good to know about the 'you cans' - I have been alternating between writing with the imperative and writing with you and it seems like the guidelines want more imperative, after reading it over. :) Thanks!